spec/01-unit/013-reports_spec.lua

```diff
@@ -55,8 +55,11 @@ describe("reports", function()
 
   describe("retrieve_redis_version()", function()
     setup(function()
-      _G.ngx = ngx
-      _G.ngx.log = function() return end
+      stub(ngx, "log")
+    end)
+
+    teardown(function(),
```

Comment:
Have you seen the linting failure and error related to this typo?

Reply:
Apologies, I need to not submit PRs before lunch :p